resolver/SuggestPackage.cc

```diff
@@ -291,6 +291,16 @@ bool SuggestPackage::tryPackageCorrections(core::Context ctx, core::ErrorBuilder
         for (auto match : missingImports) {
             pkgCtx.addMissingImportSuggestions(e, match);
         }
+
+        // going from an UnresolvedConstantLit to the full
+        // (with-scope) string is actually kinda annoying, so we're
+        // using the `loc` to get the original chunk of the file that
+        // includes the whole constant with scope
+        if (auto fullConstant = ctx.locAt(unresolved.loc).source(ctx)) {
+            // TODO (document me)
+            e.setHeader("No import provides `{}`", *fullConstant);
```

Comment:
should we add a comment somewhere that mentions that these `setHeader`'s actually overwrite any previously computed header?

Reply:
Added such a comment!